pkg/moov/transfer_models.go

```diff
@@ -371,6 +372,28 @@ type TransferOptions struct {
 	DestinationOptions []PaymentMethod `json:"destinationOptions,omitempty"`
 }
 
+// LineItems contains Items for nested structure
+type LineItems struct {
```

Comment:
could we prefix this with `CreateTransferLineItems`?

Similar comment for all the new top-level structs below too